agent/command/s3_put.go

```diff
@@ -422,6 +424,12 @@ retryLoop:
 						return errors.Wrapf(err, "checking if file '%s' exists", remoteName)
 					}
 					if exists {
+						if s3pc.preservePath {
```

Comment:
nit -- since we aren't using this value anyway, can we just keep a skippedFilesCount instead? Then we don't need an if/else here. (Seems like we might be able to do that real quick for uploadedFiles too if you want to make them consistent, but I think it's fine either way)

Reply:
We can't make that change for uploadedFiles because they are used by the attachFiles function, but I could for skippedFiles quickly!